crux-agent/investigate_stop_token_issue.py

```
@@ -0,0 +1,214 @@
+#!/usr/bin/env python3
+"""
+Investigation script to capture raw evaluation_prompt and model replies
+to understand why '<stop>' appears in LLM outputs, triggering premature convergence.
+"""
+import asyncio
+import logging
+import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)